content/languages/groovy/spock.md

```diff
@@ -7,6 +7,30 @@ tags:
 
 # Spock
 
+## Basic Setup
+
+### Solution
+
+```groovy
+class Adder {
+  static def add(a, b) { a + b }
+}
+```
+
+### Tests
+
+```groovy
+import spock.lang.Specification
+
+class AdderSpec extends Specification {
+  def "Adder.add returns the sum"() {
+    expect:
```

Comment:
What is this label for?

Reply:
Spock has that in their docs: https://spockframework.org/spock/docs/2.1/data_driven_testing.html. I guess the idea is to separate different parts of the test case. Would you recommend removing it?

Reply:
I'll resolve since the linked docs also have `expect:` labels pointing to basic assertions, but if a Kotlin expert arrives happy to have them PR a better version down the line. I'm mostly trying to get something minimal that's reasonably in the ballpark for each lang.